pkg/storegateway/bucket.go

```diff
@@ -2286,18 +2288,20 @@ func (it *bigEndianPostings) length() int {
 	return len(it.list) / 4
 }
 
-func (r *bucketIndexReader) PreloadSeries(ctx context.Context, ids []storage.SeriesRef) error {
+func (r *bucketIndexReader) PreloadSeries(ctx context.Context, ids []storage.SeriesRef, stats *safeQueryStats) (*bucketIndexLoadedSeries, error) {
```

Comment:
Nit: this shouldn't be an exporter method anymore, as it's now clear that it can only be implemented in this package.

Reply:
Done here: https://github.com/grafana/mimir/pull/3396/commits/91bea6cf60cee441dfea5475383c119cf088c875